addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -884,8 +884,9 @@ body.editor_enable.editor_has_snippets {
         }
 
         // Buttons with an icon but no text, so no possible overflow
-        we-button.o_we_button_icon {
+        we-button.fa {
```

Comment:
@younn-o Can you extract this to a separate PR?
It seems like multiple PR need it. So we can merge it quickly.

For the reminder, it was added with https://github.com/odoo/odoo/pull/73241/files#diff-f1acdd04893b739b62e26ae33945d012c4ce8df627f23ea84968f707369c8eaeR887
Being improved here (automatic instead of class required).
But also needed for #74206

Reply:
I extracted it here : https://github.com/odoo/odoo/pull/77173